templates/students/add_new_student.html

```diff
@@ -0,0 +1,117 @@
+{% extends 'layout.html' %} {% block style %}
+<style>
+  /* .container {
+            max-width: 960px;
+        } */
+
+  @media screen and (max-width: 600px) {
+    .container {
+      width: 85vw;
+    }
+  }
+</style>
+{% endblock %} {% block content %}
+
+<div class="container">
+  <div class="py-3 mt-3 text-center">
+    <h2 class="font-weight-bolder">Add new student</h2>
+  </div>
+
+  <div class="row justify-content-center">
+    <div class="col-md-8 shadow-lg p-5 bg-white rounded">
+      <form method="POST" enctype="multipart/form-data" class="needs-validation" novalidate>
+        {% csrf_token %}
+
+        <div class="row">
+          <div class="col-md-6 mb-3">
+            <label for="firstName">First name</label>
+            <input name="first_name" type="text" class="form-control" id="firstName" placeholder="" value="" required />
+            <div class="invalid-feedback">Valid First Name is required.</div>
+          </div>
+          <div class="col-md-6 mb-3">
+            <label for="lastName">Last name</label>
+            <input name="last_name" type="text" class="form-control" id="lastName" placeholder="" value="" required />
+            <div class="invalid-feedback">Valid Last Name is required.</div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-md-4 mb-3">
+            <label for="gender">Gender</label>
+            <select name="gender" class="custom-select d-block w-100" id="gender" required>
+              <option value="M">Male</option>
+              <option value="F">Female</option>
+              <option value="O">Other</option>
+            </select>
+          </div>
+          <div class="col-md-4 mb-3">
+            <label for="school_class">Class</label>
+            <input name="school_class" type="number" class="form-control" id="school_class" placeholder="8" min="1" max="12" required />
```

Comment:
Don't put any bar on the class. We also have 13 and 14 classes to represent students preparing for Navodaya.

Reply:
You haven't removed the restrictions yet from `school_class`.